api/create.go

```diff
@@ -44,7 +44,7 @@ func CreateService(w http.ResponseWriter, req *http.Request) {
 	}
 
 	err = json.Unmarshal(byteArray, &s)
-	log.Printf("%d %d", s.Version.Maj, s.Version.Min)
+	log.Printf("%d %d %s %s", s.Version.Maj, s.Version.Min, s.Db.CPU, s.Db.Memory)
```

Comment:
looks like this hasn't been changed to `%d` when you changed the type to `int64`.

Reply:
also, wondering how we can catch this? As part of our CI build add a test? cc @idoqo

Reply:
yeah, this kind of things are usually handled by linters, so we could add one to our CI (and for local dev too).